### PR TITLE
ruby example to create themes

### PR DESCRIPTION
**Pull request by DucarrougeR**
Adding an example to create themes in Looker instance using Ruby SDK

### CODE REVIEW

File: examples/ruby/README.md [hunk 1]
```diff
@@ -9,38 +9,38 @@ Using official Looker Ruby SDK: https://github.com/looker/looker-sdk-ruby/
...
 ___________
 ### USERS 👤
-- Logout all users on the instance [[link]](logout_all_users.rb)
-- Disable users in the instance [[link]](disable_users.rb)
-- Get a list of all users and their auth credentials [[link]](users_list_and_auth_types.rb)
-- Add all users to a group [[link]](all_users_to_group.rb)
+- Logout all users on the instance [[link]](https://github.com/looker-open-source/sdk-examples/blob/master/ruby/logout_all_users.rb)
+- Disable users in the instance [[link]](https://github.com/looker-open-source/sdk-examples/blob/master/ruby/disable_users.rb)
+- Get a list of all users and their auth credentials [[link]](https://github.com/looker-open-source/sdk-examples/blob/master/ruby/users_list_and_auth_types.rb)
+- Add all users to a group [[link]](https://github.com/looker-open-source/sdk-examples/blob/master/ruby/all_users_to_group.rb)
```
Comment (jkaster) [inline examples/ruby/README.md:15]: Are these links not resolving when left relative? Fully qualifying the URL shouldn't be required, and isn't the right pattern, IMO.

File: examples/ruby/create_themes.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+strava_like = {:name=>"strava", :settings=>{:background_color=>"#fff8f5", :base_font_size=>"12px", :color_collection_id=>"7c79334a-9912-4ca1-be6a-35756782ae09", :font_color=>"#FC4C02", :font_family=>"Arial", :font_source=>"", :info_button_color=>"#FC4C02", :primary_button_color=>"#FC4C02", :show_filters_bar=>true, :show_title=>true, :text_tile_text_color=>"", :tile_background_color=>"#fff8f5", :tile_text_color=>"#FC4C02", :title_color=>"#FC4C02", :warn_button_color=>"#FC4C02"}}
+sendgrid_like =  {:name=>"sendgrid", :settings=>{:background_color=>"#f6f6f6", :base_font_size=>"12px", :color_collection_id=>"1bc1f1d8-7461-4bfd-8c3b-424b924287b5", :font_color=>"#1A82e2", :font_family=>"Arial", :font_source=>"", :info_button_color=>"#294661", :primary_button_color=>"#294661", :show_filters_bar=>true, :show_title=>true, :text_tile_text_color=>"", :tile_background_color=>"#f6f6f6", :tile_text_color=>"#1A82e2", :title_color=>"#1A82e2", :warn_button_color=>"#294661"}}
```
Comment (jkaster) [inline examples/ruby/create_themes.rb:44]: Nice! Cool idea.

**Review by jkaster — CHANGES_REQUESTED** [review]
Please make the doc links relative.

File: examples/ruby/README.md (hunk 1 above)
Comment (DucarrougeR) [reply examples/ruby/README.md]: Updated the README with relative links now. Thanks for the feedback John!

**Review by jkaster — APPROVED** [review]
LGTM

File: examples/ruby/create_themes.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+# 		    :text_tile_text_color=> "",
+# 		    :tile_background_color=> "#efaa3b",
+# 		    :tile_text_color=> "#3a4245",
+# 		    :title_color=> "#7d0f48",
+# 		    :warn_button_color=> "#980c11"
+# 	  }
+# 	}
+​
+# these are general examples and not necessarily themes that follow the brand guidelines for these companies
+airbnb_like = {:name=>"airbnb", :settings=>{:background_color=>"#faf3f2", :base_font_size=>"12px", :color_collection_id=>"5f313589-67ce-44ba-b084-ec5107a7bb7e", :font_color=>"#143f85", :font_family=>"Arial", :font_source=>"", :info_button_color=>"#0087e1", :primary_button_color=>"#FF5A5F", :show_filters_bar=>true, :show_title=>true, :text_tile_text_color=>"", :tile_background_color=>"#faf3f2", :tile_text_color=>"#FF5A5F", :title_color=>"#FF5A5F", :warn_button_color=>"#980c11"}}
```
Comment (joeldodge79) [inline examples/ruby/create_themes.rb:37]: sorry, late comment here but consider reformatting these for a/ new-style symbol keys and b/ vertical readability:

```ruby
airbnb_like = {
  name: "airbnb",
  settings: {
    background_color: "#faf3f2",
    base_font_size: "12px",
    color_collection_id: "5f313589-67ce-44ba-b084-ec5107a7bb7e",
    font_color: "#143f85",
    font_family: "Arial",
    font_source: "",
    info_button_color: "#0087e1",
    primary_button_color: "#FF5A5F",
    show_filters_bar: true,
    show_title: true,
    text_tile_text_color: "",
    tile_background_color: "#faf3f2",
    tile_text_color: "#FF5A5F",
    title_color: "#FF5A5F",
    warn_button_color: "#980c11",
  },
}
```

File: examples/ruby/create_themes.rb [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+sendgrid_like =  {:name=>"sendgrid", :settings=>{:background_color=>"#f6f6f6", :base_font_size=>"12px", :color_collection_id=>"1bc1f1d8-7461-4bfd-8c3b-424b924287b5", :font_color=>"#1A82e2", :font_family=>"Arial", :font_source=>"", :info_button_color=>"#294661", :primary_button_color=>"#294661", :show_filters_bar=>true, :show_title=>true, :text_tile_text_color=>"", :tile_background_color=>"#f6f6f6", :tile_text_color=>"#1A82e2", :title_color=>"#1A82e2", :warn_button_color=>"#294661"}}
+​
+all_themes = [airbnb_like, gcp_like, aws_like, microsoft_like, facebook_like, datadog_like, strava_like, sendgrid_like]
+​
+# pull the list of themes we created above and create them all in our instance
+all_themes.each { | theme|
+	sdk.create_theme(theme)
+}
```
Comment (joeldodge79) [inline examples/ruby/create_themes.rb:51]: extra space in `|  theme|` - I think we generally don't pad these

Also, (and I had to look this up) we stick to [this convention](https://stackoverflow.com/questions/5587264/do-end-vs-curly-braces-for-blocks-in-ruby)

so you should either
```ruby
all_themes.each { |theme| sdk.create_theme(theme) }
```
or
```ruby
all_themes.each do |theme|
  sdk.create_theme(theme)
end
```

**Comment by joeldodge79** [comment]
nice work Romain! thanks for contributing. I left some stylistic nit-pics that computers should really just be solving for us these days (like js Prettier or python Black). Maybe I'll experiment with something like https://github.com/prettier/plugin-ruby for this examples/ruby/ directory :-)

**Comment by jkaster** [comment]
Ruby reformatting would be great

**Pull request merged**